src/langchain_google_alloydb_pg/vectorstore.py

```diff
@@ -773,7 +773,7 @@ async def amax_marginal_relevance_search_with_score_by_vector(
         fetch_k = fetch_k if fetch_k else self.fetch_k
         lambda_mult = lambda_mult if lambda_mult else self.lambda_mult
         embedding_list = [json.loads(row[self.embedding_column]) for row in results]
-        mmr_selected = maximal_marginal_relevance(
+        mmr_selected = utils.maximal_marginal_relevance(
```

Comment:
Please update pyproject.toml to set the minimum version to  0.2.36

Reply:
This change has been made now.